src/settings/components/MCLPaginationFields.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { Field } from 'react-final-form-html5-validation';
+
+import {
+  Col,
+  Layout,
+  Row,
+  TextField,
+} from '@folio/stripes/components';
+
+const lists = ['agreementLines', 'agreementEresources', 'entitlementOptions', 'packageContents', 'entitlements'];
+
+const inRange = (x, min, max) => {
+  return ((x - min) * (x - max) <= 0);
+};
+
+const validate = (fieldValue, min, max) => {
+  return (!inRange(fieldValue, min, max)) ?
+    <FormattedMessage id="ui-agreements.settings.error.valueNotInRange" values={{ min, max }} /> : undefined;
+};
+
+const MCLPaginationFields = () => {
+  return (
+    <>
+      <strong>
+        <Layout className="padding-bottom-gutter padding-top-gutter" data-test-mcl-description>
+          <FormattedMessage id="ui-agreements.settings.mcl.pagination.description" />
```

Comment:
If you need the `description` here to be strong, you could pass it in the `translations` file and take off the `<strong>` wrapper here?

Reply:
how can I do this with `FormattedMessage`? I tried `strong` as well as `b` tags, both are not working